### PR TITLE
new course external apis: adding google streetview and directions lessons 

### PR DESCRIPTION
**Pull request by michellechandra**
@ohasselblad

### CODE REVIEW

**Comment by michellechandra** [comment]
I am hosting the "starter file" in a gist, so that would need to be created and hosted by CartoDB (first sentence under "Getting Started": First, copy the HTML starter code stored here to a file in the text editor of your choice. We will be adding code to this file throughout the tutorial.)

Also, the final example map needs to be added to the page using CartoDB's Google account API key.

File: _app/index.html [hunk 1]
```diff
@@ -115,6 +115,44 @@ <h3 class="Feature-title">{{ collection_item.title }}</h3>
         </div>
       </div>
     </section>
+
+    <section id="advanced" class="Grid">
```
Comment (ohasselblad) [inline _app/index.html]: @carlostallon I added a new section but the margins on top appear different than the one between the `easy` and `medium` sections. Could you take a look please?

Comment (ohasselblad) [inline _app/index.html]: Actually, just fixed it, never mind!

**Comment by ohasselblad** [comment]
@michellechandra, could you add some references on the bottom of the page to sum up the major resources and concepts covered? Also, should the final, working version be linked at the bottom?

**Comment by ohasselblad** [comment]
@vuxita we need a new mailchimp id for this course (it's a new series of lessons). Please let me know what you need from me!

**Comment by ohasselblad** [comment]
@matallo, the live/working example of this uses CartoDB's Google API key.. How should we handle hosting this example?

**Comment by matallo** [comment]
arg, I did this with @javisantana some time ago, but can't remember what he did exactly (maybe I'm mixing things), can you help us Javi?

**Comment by ohasselblad** [comment]
bump bump, any advice would be appreciated!

**Comment by csobier** [comment]
Can I get a status update for what the hold up is? I need to reference this published tutorial as part of a Docs feature for the next Sprint milestone.

**Comment by matallo** [comment]
forwarded this to @danicarrion to see if they can help from @CartoDB/solutions

**Comment by ohasselblad** [comment]
ok, @michellechandra, we have a solution! We're going to set up the browser key with a domain filter so that it can only be used coming from academy.cartodb.com. See me when you have some time and we'll get it set up together.

**Comment by ohasselblad** [comment]
Looks great, let's merge :)

**Pull request merged**